src/common/components/contactInfo/contactInfo.tpl.html

```diff
@@ -220,5 +220,24 @@ <h4 class="panel-title  border-bottom-small  visible" translate>{{'CONTACT_INFO'
         </div>
       </div>
     </div>
+
+    <div class="mb" ng-if="$ctrl.requestRadioStation">
+      <h4 class="panel-title  border-bottom-small  visible" translate>{{'RADIO_STATION'}}</h4>
+      <div class="form-group">
+        <select class="form-control form-control-subtle"
+          name="radioStation"
+          ng-model="$ctrl.radioStation"
+          ng-options="v.Description as v['Description'] for v in $ctrl.radioStations | orderBy:'\'Description\''"
```

Comment:
`orderBy:'\'Description\''` seems a bit weird to me. Do we need the quotes in the text for some reason?

Reply:
Turns out they're not needed!  I'll clean that up.